### PR TITLE
Pari seadata and seadata-big , fix wildcard on vcopy

### PR DESCRIPTION
**Pull request by motorto**

### CODE REVIEW

**Comment by sgn** [comment]
So they need to be placed inside `/usr/share/pari` instead of `/usr/share/pari/seadata`? If yes, all other `pari-*` needs to be fixed too.

File: srcpkgs/pari-seadata/template [hunk 1]
```diff
@@ -13,5 +13,5 @@ checksum=c9282a525ea3f92c1f9c6c69e37ac5a87b48fb9ccd943cfd7c881a3851195833
 
 do_install() {
  vmkdir usr/share/pari/seadata
- vcopy "data/seadata/sea???" usr/share/pari/seadata
+ vcopy data/seadata/sea* usr/share/pari/seadata
```
Comment (sgn) [inline srcpkgs/pari-seadata/template:16]: I suggested this because it's about to split `seadata` and `seadata-small`.

**Comment by dkwo** [comment]
Thank you, I tested seadata and now it works.

**Comment by motorto** [comment]
> So they need to be placed inside `/usr/share/pari` instead of `/usr/share/pari/seadata`? If yes, all other `pari-*` needs to be fixed too.

Yes @sgn , that was the problem, (and the ??? ) , well I tried the other packages with the test upstream offers , and they all work.

> D. To test whether a package is available.
> 
> Here are sample commands that will fail if a given package is not available:
> 
>   ellinit("11a1")     /* package: elldata */
>   galoisgetpol(8)     /* package: galpol */
>   galoisgetname(12,1) /* package: galpol 4.0 and up */
>   ellmodulareqn(11)   /* package: seadata */
>   polgalois(x^8-2)    /* package: galdata */

https://pari.math.u-bordeaux.fr/packages.html

**Pull request merged**